### ISSUE TITLE
Availability of container/kliko in Cranendonck

### ISSUE THREAD

**Issue by Miicr0** [pippyn/Home-Assistant-Sensor-Afvalbeheer#291]
The waste collectors in the municipality of Cranendonck make a distinction between trashbags (tariefzak rest) and gray containers (container restafval). The current sensor detects the resource 'restafval' as tariefzak rest. I've tried multiple ways to get insight in the container, but have not been able to map this.  Is there a way to obtain information on the container restafval?

printwastetypes: 1 returns the following: 
Available waste types: gft, papier, pmd, restafval
Making me assume that the container is not yet supported.

**Comment by pippyn** [comment]
Can you give me an address to test with?

**Comment by Miicr0** [comment]
Certainly, one that should work is: 6026EG, 1Op 18 jan. 2022 08:19 schreef Pippyn ***@***.***>:
Can you give me an address to test with?

**Comment by pippyn** [comment]
Fixed in v4.9.2

Example config:
```
sensor:
  - platform: afvalbeheer
    wastecollector: cranendonck
    resources:
      - gft
      - papier
      - pmd
      - restafval
      - restafvalzakken
    postcode: 6026EG
    streetnumber: 1
    printwastetypes: 1
```

**Comment by Miicr0** [comment]
Thank you, it works!
I'll close the issue.
